recipes/libsamplerate/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "0.1.8" %}
+
+package:
+  name: libsamplerate
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/erikd/libsamplerate.git
+  git_tag: 548c6761a0348a83c5f6ee68aa1581c855092195
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - autoconf # [not win]
+    - automake # [not win]
+    - libtool # [not win]
+    - pkg-config # [not win]
+
+test:
+  commands:
+    - true  # we perform testing in the build phase
+
+about:
+  home: http://www.mega-nerd.com/SRC/
+  license: Dual (GPL + commercial)
```

Comment:
This license is pretty fun. Could we know what GPL this is? Also, what is the deal with the commercial license?